pkg/utils/tp_proc_test.go

```diff
@@ -43,6 +54,35 @@ func TestCalcTPOrderShortNoFees(t *testing.T) {
 	assert.NotEmpty(t, order.ClientOrderID)
 }
 
+func TestCalcTPOrderShortNoFees2(t *testing.T) {
```

Comment:
Add a description of the difference between these two cases in the function name